src/components/Avatar/Avatar.tsx

```diff
@@ -22,19 +35,36 @@ function Avatar({ onLogOut }: AvatarProps) {
       ? anchorRef.current.getBoundingClientRect()
       : null;
 
+   const avatarStyle = user?.avatar
+      ? {
+           backgroundImage: `url(${user.avatar})`,
+           backgroundSize: 'cover',
+           backgroundPosition: 'center',
+        }
+      : undefined;
+
+   const initials = user
+      ? user.name
+           .split(' ')
+           .map((n) => n[0])
+           .join('')
```

Comment:
It takes the initial in case of no image

Reply:
> It takes the initial in case of no image

However, it goes a long, complex way to achieve that.
```
user.name // canopee
  .split(' ') // ['canopee']
  .map(n => n[0]) // 'canpoee' --> 'c' --> [ ] --> ['c']
  .join('') // 'c'

---

user.name[0] // 'c'

---

user.name.slice(0, 1) // 'c'
```